src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -97,13 +97,17 @@ def load_arguments(self, _):
 
     with self.argument_context('backup vault create') as c:
         c.argument('tags', arg_type=tags_type)
+        c.argument('classic_alerts', arg_type=get_enum_type(['Enable', 'Disable']), help='Use this property to specify whether backup alerts from the classic solution should be received.')
+        c.argument('azure_monitor_alerts_for_job_failures', options_list=['--alerts-for-job-failures'], arg_type=get_enum_type(['Enable', 'Disable']), help='Use this property to specify whether built-in Azure Monitor alerts should be received for every job failure.')
```

Comment:
Could we define those parameter as `arg_type=get_three_state_flag()`? In this way, you can convert them into `Enabled` or `Disabled` according to `True` or `False`

Reply:
Hey! Enable/Disable looks like a better choice for us as the same `az backup vault backup-properties set` command has other parameters like `--soft-delete-feature-state` and `--hybrid-backup-security-features` which take Enable/Disable as inputs and we wanted to keep the experience consistent. 

Also, it feels more natural to Enable/Disable alerts rather than setting them to True/False.

Reply:
OK, make sense